printer-linter/src/terminal.py

```diff
@@ -74,6 +75,18 @@ def formatFile(file: Path, settings) -> None:
             config.write(f, space_around_delimiters=settings["format"].get("format-profile-space-around-delimiters", True))
 
 
+def extract_file_paths(paths: List[Path]) -> List[Path]:
```

Comment:
I hate to say this but, not according to UM codestyle convention
```suggestion
def extractFilePaths(paths: List[Path]) -> List[Path]:
```